### PR TITLE
`getblock` optionally returns txns count rather than txns list

### PR DESCRIPTION
**Pull request by sickpig**

### CODE REVIEW

**Comment by sickpig** [comment]
unintentionally committed a travis change, will revert as soon as I get home.

File: src/rpc/blockchain.cpp [hunk 1]
```diff
@@ -363,15 +371,17 @@ UniValue getblockheader(const UniValue& params, bool fHelp)
...
+            "getblock \"hash\" ( verbose ) ( listtransactions )\n"
             "\nIf verbose is false, returns a string that is serialized, hex-encoded data for block 'hash'.\n"
             "If verbose is true, returns an Object with information about block <hash>.\n"
+            "If listtransactions is true, a list of the IDs of all the transactions included in the block will be shown.\n"
             "\nArguments:\n"
             "1. \"hash\"          (string, required) The block hash or height\n"
             "2. verbose           (boolean, optional, default=true) true for a json object, false for the hex encoded data\n"
-            "\nResult (for verbose = true):\n"
+            "3. listtransactions  (boolean, optional, default=true) true to get a list of all txns, false to get just txns count\n"
+            "\nResult (for verbose = true, listtransactions = true):\n"
```
Comment (ptschip) [inline src/rpc/blockchain.cpp:384]: nit:  perhaps change this to say "...  false to get a count only"

**Review by ptschip — APPROVED** [review]
Untested but looks good to me.

File: src/rpc/blockchain.cpp (hunk 1 above)
Comment (sickpig) [reply src/rpc/blockchain.cpp:384]: think your wording is better, going to change.

**Comment by gandrewstone** [comment]
we've been actively using this change on the giga_perf branch, so it is tested, merging. thanks!

**Pull request merged**